google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/CloudSdkAppEngineHelper.java

```diff
@@ -137,9 +138,10 @@ public CancellableRunnable createDeployRunner(
         wrapCallbackForUsageTracking(
             callback, deploymentConfiguration, source.getFile(), targetEnvironment));
 
-    if (targetEnvironment == AppEngineEnvironment.APP_ENGINE_STANDARD) {
+    if (targetEnvironment.isStandard()
+        || (targetEnvironment.isFlexible() && AppEngineUtil.isFlexCompat(project, source))) {
```

Comment:
for frameworks, I'd say there's just 'standard' and 'non-standard'.

On Mon, Jun 13, 2016 at 5:31 PM, Etan Shaul <notifications@github.com>
wrote:

> In
> google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/CloudSdkAppEngineHelper.java
> <https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/728#discussion_r66872239>
> :
>
> > @@ -137,9 +138,10 @@ public CancellableRunnable createDeployRunner(
> >          wrapCallbackForUsageTracking(
> >              callback, deploymentConfiguration, source.getFile(), targetEnvironment));
> >
> > -    if (targetEnvironment == AppEngineEnvironment.APP_ENGINE_STANDARD) {
> > +    if (targetEnvironment.isStandard()
> > +        || (targetEnvironment.isFlexible() && AppEngineUtil.isFlexCompat(project, source))) {
>
> yea we really do. Now that you mention it - if AppEngineEnvironment wasn't
> thought of as the target cloud environment, but instead we thought of it as
> the 'framework type', then possibly my code prior to these last changes
> would make more sense:
>
> 3 framework types, 'standard', 'flexible', 'flex-compat' (maybe even
> 'standard-compat' would make more sense).
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/728/files/0f02d0ae3fe9c9f17db25a3d0cf45fd859f96bfc#r66872239>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AHf5HXWvx6tEtul3wVeLDr2NJF6rxnu4ks5qLcwUgaJpZM4I0kNE>
> .
>